flask_app/migrations/2025_01_30_152951_fix_bioreplicates_metadata_unique_id.py

```diff
@@ -0,0 +1,26 @@
+import sqlalchemy as sql
+
+def up(conn):
+    query = """
+        ALTER TABLE BioReplicatesMetadata
+        DROP PRIMARY KEY,
+        ADD PRIMARY key(studyId, bioreplicateUniqueId);
+    """
```

Comment:
The `(studyId, bioreplicateUniqueId)` pair is already unique, while `bioreplicateId` is not. But the primary key is, for some reason, the second, and not the first. This migration fixes that, allowing for two different experiments to have the same name for a bioreplicate.